server/Assignments_filler/sheetFiller.py

```diff
@@ -0,0 +1,237 @@
+from __future__ import print_function
+
+import os.path
+import logging
+import sys
+import time
+
+from DB_call import get_from_db
+from googleapiclient.discovery import build
+from google_auth_oauthlib.flow import InstalledAppFlow
+from google.auth.transport.requests import Request
+from google.oauth2.credentials import Credentials
+
+# If modifying these scopes, delete the file token.json.
+SCOPES = ['https://www.googleapis.com/auth/spreadsheets']
+
+# The ID and range of a sample spreadsheet.
+SPREADSHEET_ID = '1W-C0ozCqXaphvQe1Cz1IFPYwvgO_MXpn1BLKcdI-n6o'
+TEAM_ASSINGMENT_TAB = 'Team Assignments!'
+RANGE_READ = 'F33'
+refreshRate = 10
+
+logging.basicConfig(level=logging.INFO, format="%(asctime)s [%(levelname)s]: %(message)s",
+                    handlers=[logging.FileHandler("filler.log"), logging.StreamHandler()])
+
+def readSheet(service, range, amount = 1):
```

Comment:
```suggestion
def readSheet(service, range, amount=1):
```
Arguments have no spaces between name and value.

Reply:
Why did you revert this!?

Reply:
By mistake. It's already fixed